tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -1989,6 +1989,10 @@ public function sendShareSpaceRequest(
 		string $role
 	): void {
 		$space = $this->getSpaceByName($user, $spaceName);
+		$availableRoleToAssignToShareSpace = ['manager','editor','viewer'];
+		if (!\in_array($role, $availableRoleToAssignToShareSpace)) {
```

Comment:
make case-insensitive? If yes, save lowercase $role
```suggestion
		if (!\in_array(\strtolower($role), $availableRoleToAssignToShareSpace)) {
```